### PR TITLE
Add Axe Kick animation

### PR DESCRIPTION
**Pull request by kittenchilly**
## Description
Fairly simple Axe Kick animation, made by me

## Images
axekick

## **Discord contact info**
kittenchilly

### CODE REVIEW

**Comment by hedara90** [comment]
The delay between squish and unsquish feels a bit long. Probably because what the animation is used for in other places needed it.
It can be shortened by making a new `static const union AffineAnimCmd sSquishTargetAffineAnimCmds` for this animation.

https://github.com/user-attachments/assets/e112915b-547c-4136-bf31-9af79120cc21

File: data/battle_anim_scripts.s [hunk 1]
```diff
@@ -17844,8 +17844,22 @@ ElectroShotUnleash:
 	blendoff
 	end
 
-Move_TERA_BLAST::
 Move_AXE_KICK::
+	loadspritegfx ANIM_TAG_HANDS_AND_FEET
+	loadspritegfx ANIM_TAG_IMPACT
+	playsewithpan SE_M_JUMP_KICK, SOUND_PAN_ATTACKER
+	createsprite gSlideMonToOffsetSpriteTemplate, ANIM_ATTACKER, 2, 0, 20, 0, 0, 4
+	delay 0x7
```
Comment (hedara90) [inline data/battle_anim_scripts.s:17852]:
```suggestion
	delay 7
```
Use decimal numbers unless there's a good reason for hex

File: data/battle_anim_scripts.s [hunk 2]
```diff
@@ -17844,8 +17844,22 @@ ElectroShotUnleash:
...
+	createsprite gAxeKickSpriteTemplate, ANIM_TARGET, 3
+	delay 0x2
```
Comment (hedara90) [inline data/battle_anim_scripts.s:17854]:
```suggestion
	delay 2
```
Use decimal numbers unless there's a good reason for hex

File: data/battle_anim_scripts.s [hunk 3]
```diff
@@ -17844,8 +17844,22 @@ ElectroShotUnleash:
...
+	playsewithpan SE_M_COMET_PUNCH, SOUND_PAN_TARGET
+	createsprite gBasicHitSplatSpriteTemplate, ANIM_ATTACKER, 1, 0, 0, ANIM_TARGET, 1
+	createvisualtask AnimTask_SquishTarget, 0x2
```
Comment (hedara90) [inline data/battle_anim_scripts.s:17857]:
```suggestion
	createvisualtask AnimTask_SquishTarget, 2
```
Use decimal numbers unless there's a good reason for hex

**Review by hedara90 — CHANGES_REQUESTED** [review]
hex to dec

**Review by Brandon-M-Hartman — APPROVED** [review]

**Review by Brandon-M-Hartman — APPROVED** [review]

**Comment by kittenchilly** [comment]
Alright, reviews addressed and I shortened the squish animation

**Review by hedara90 — APPROVED** [review]

**Pull request merged**